includes/class-fs-api.php

```diff
@@ -226,26 +228,49 @@ private function _call( $path, $method = 'GET', $params = array(), $retry = fals
 
                 if ( null !== $result &&
                      isset( $result->error ) &&
-                     isset( $result->error->code ) &&
-                     'request_expired' === $result->error->code
+                     isset( $result->error->code )
                 ) {
-                    if ( ! $retry ) {
-                        $diff = isset( $result->error->timestamp ) ?
-                            ( time() - strtotime( $result->error->timestamp ) ) :
-                            false;
-
-                        // Try to sync clock diff.
-                        if ( false !== $this->_sync_clock_diff( $diff ) ) {
-                            // Retry call with new synced clock.
-                            return $this->_call( $path, $method, $params, true );
+                    $retry = false;
+
+                    if ( ! $in_retry ) {

```

Comment:
I believe this criterion can be moved to the parent `if` (the one starting with `if ( null !== $result &&`), which will remove the unnecessary indentation. It now makes more sense due to the extra `if ( $retry ) {` below, which never gets executed if `$in_retry` is `true`.